test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua

```diff
@@ -0,0 +1,119 @@
+--  Requirement summary:
+--  [HeartBeat]: SDL must close only session in case mobile app does not answer on Heartbeat_request
+--  [HeartBeat] [GENIVI]: SDL must start HeartBeat process immediately after first StartService request from mobile app
+--
+--  Description:
+--  Check that heartbeat occurs if App uses v3 protocol version and doesn't send HB to SDL
```

Comment:
@LitvinenkoIra, may be will be better if you write heartbeat timeout, instead only heartbeat.